packages/expo/src/plugin/getErrorTrackingPluginsFromOptions.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import type { ConfigPlugin, StaticPlugin } from '@expo/config-plugins';
+
+import { GradlePluginDatadogSite } from './pluginGlobalConfiguration';
+import withAndroidProguardMappingFiles from './withAndroidProguardMappingFiles/withAndroidProguardMappingFiles';
+import withAndroidSourcemaps from './withAndroidSourcemaps/withAndroidSourcemaps';
+import withIosDsyms from './withIosDsyms/withIosDsyms';
+import withIosSourcemaps from './withIosSourcemaps/withIosSourcemaps';
+
+export type ErrorTrackingOptions = {
+    iosDsyms?: boolean;
+    iosSourcemaps?: boolean;
+    androidProguardMappingFiles?: boolean;
+    androidSourcemaps?: boolean;
+};
+
+/**
+ * By default, all plugins are enabled. To disable a plugin, you have to set it
+ * to `false`.
+ */
+export const getErrorTrackingPluginsFromOptions = (
+    options: ErrorTrackingOptions | void,
```

Comment:
in which case it can be `void`?

Reply:
If the user does not specify any option in the config.
If they want to send all the mapping files, they don't have to specify a config, every file is sent by default.

Reply:
```suggestion
    options: ErrorTrackingOptions | undefined,
```
`void` is typically used to reference an empty return not an input.  `undefined` would be a better choice. Ideally, it would be `options?` instead, but that would require `params` to be optional as well.

Reply:
Now that we don't need params anymore, I can change it to `options?`. Thanks for pointing this out!